docs/source/conf.py

```diff
@@ -36,4 +36,9 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
 html_theme = "sphinx_rtd_theme"
```

Comment:
What do you think about using [furo theme](https://sphinx-themes.org/sample-sites/furo/)?

```suggestion
html_theme = "furo"
```
![imagen](https://github.com/DashAISoftware/DashAI/assets/22802485/6419b3f8-618a-4b83-b8e7-85a477fe1486)